code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -133,7 +133,7 @@
 		list("Sentry Gun Network Laptop", 4, /obj/item/device/sentry_computer, VENDOR_ITEM_REGULAR),
 		list("JTAC Pamphlet", round(scale * 1), /obj/item/pamphlet/skill/jtac, VENDOR_ITEM_REGULAR),
 		list("Engineering Pamphlet", round(scale * 1), /obj/item/pamphlet/skill/engineer, VENDOR_ITEM_REGULAR),
-		list("Powerloader Certification", round(scale * 0.1), /obj/item/pamphlet/skill/powerloader, VENDOR_ITEM_REGULAR),
+		list("Powerloader Certification", round(scale * 1), /obj/item/pamphlet/skill/powerloader, VENDOR_ITEM_REGULAR),
```

Comment:
Do (round(scale * 0.1) + 1) so as to guarantee at least 1 but not totally upset the amount we generally want in a higher pop round.